README.md

```diff
@@ -32,14 +32,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+A programming object that represents html elements of a webpage. 
+
 2. What is an event?
 
+Things that happen to the html elements.
+
 3. What is an event listener?
 
+A method on an element that will take two arguments, first one to listen to, and second is the callback for when the event is triggered.
+
 4. Why would we convert a NodeList into an Array?
 
+The NodeList will create a new Array from an existing one.
```

Comment:
No, the NodeList does _not_ create a new array from an existing array. 

The question is why would we convert a NodeList into an array. 

The reason is so that we have access to all of the array methods arrays have, such as .map and .filter.